docs/standard-library/ios-typedefs.md

```diff
@@ -61,7 +61,7 @@ int main( )
    ofstream x( "iostream.txt" );
    x << "testing";
    streampos y = x.tellp( );
-   cout << y << endl;
+   cout << + y << endl;
```

Comment:
I don't think this example should take advantage of the fact our implementation of `fpos` happens to be implicitly convertible to `streamoff`, since the standard only requires that it can be explicitly converted (http://eel.is/c++draft/tab:fpos.operations#row-7). I also think that using unary `+` here to force that conversion is fairly arcane; it makes the example less clear rather than moreso.
Why not simply:
```suggestion
   cout << streamoff(y) << '\n';
```
(I replaced `endl` as a drive-by because we shouldn't teach it as the "default" way to end lines.)